redwood-layout-composeui/src/main/kotlin/app/cash/redwood/layout/ComposeLayoutWidgetFactory.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.layout
+
+import androidx.compose.runtime.Composable
+import app.cash.redwood.layout.widget.Column
+import app.cash.redwood.layout.widget.RedwoodLayoutWidgetFactory
+import app.cash.redwood.layout.widget.Row
+
+class ComposeLayoutWidgetFactory : RedwoodLayoutWidgetFactory<@Composable () -> Unit> {
```

Comment:
```suggestion
public class ComposeRedwoodLayoutWidgetFactory : RedwoodLayoutWidgetFactory<@Composable () -> Unit> {
```
Should have failed without the modifier.

The other type is named with "RedwoodLayout" but looks like it's file name doesn't reflect the type name.